src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -539,99 +624,46 @@ private async Task RefreshKeyValueCollections(CancellationToken cancellationToke
                 }
 
                 IEnumerable<ConfigurationSetting> currentKeyValues;
-                IEnumerable<KeyValueChange> keyValueChanges;
-                bool success = false;
+                IEnumerable<KeyValueChange> localKeyValueChanges;

```

Comment:
Looks like `localKeyValueChanges` could be removed since we can just add the result of `GetKeyValueChangeCollection` to `keyValueChanges` directly.